src/coreclr/vm/ceemain.cpp

```diff
@@ -1326,6 +1326,8 @@ void STDMETHODCALLTYPE EEShutDownHelper(BOOL fIsDllUnloading)
 
         {
             CONTRACT_VIOLATION(ModeViolation);
+            // At the moment, this doesn't do anything more than log statistics.
+            VirtualCallStubManager::UninitStatic();
```

Comment:
It would be nice to rename it to `VirtualCallStubManager::LogFinalStats` or something like that to make it clear that it is not actually uninitializing anything.